spec/02-integration/01-cmd/02-start_stop_spec.lua

```diff
@@ -1,17 +1,19 @@
 local helpers = require "spec.helpers"
 
+local KILL_ALL = "pkill nginx; pkill serf; pkill dnsmasq"
```

Comment:
Far from fifty times, and it won't change and it's tests, not actual code. But now, someone has to go up all the way to figure out what this does.